setup.py

```diff
@@ -23,11 +23,13 @@
     install_requires=[
         "rich-argparse==1.0.0",
         "rich==13.0.1",
-        "mistune==0.8.4",
         "chardet==5.1.0",
         "requests==2.31.0",
         "PyYAML==6.0.1",
         "gitignorefile==1.1.2",
+        "mistune==3.0.1",
+        "tortilla==0.5.0",
+        "gitignore_parser==0.1.6",
```

Comment:
Same for this. Maybe you forked from an older version?

Reply:
@iamjackg I did use an older version first, then I thought I fixed that.
I will corect this and try to remember why I changed the tests :D

Reply:
@iamjackg I fixed this. only updated mistune and removed dependencies that are not on the latest m2cf version